build-utils/protocol-base-mspec/src/main/java/org/apache/plc4x/plugins/codegenerator/language/mspec/model/references/DefaultIntegerTypeReference.java

```diff
@@ -23,8 +23,16 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 public class DefaultIntegerTypeReference extends DefaultSimpleTypeReference implements IntegerTypeReference {
 
-    public DefaultIntegerTypeReference(SimpleBaseType baseType, int sizeInBits) {
+    private final boolean littleEndian;
+
+    public DefaultIntegerTypeReference(SimpleBaseType baseType, int sizeInBits, boolean littleEndian) {
         super(baseType, sizeInBits);
+        this.littleEndian = littleEndian;
+    }
+
+    @Override
+    public boolean isLittleEndian() {
+        return littleEndian;
     }
```

Comment:
We could consider removing the "@Override" and leaving the build-tools repo unchanged ...